### ISSUE TITLE
Include a list of available field types in the AST?

### ISSUE THREAD

**Issue by timriley** [icelab/formalist#11]
@makenosound would a form renderer want to know something like this, e.g. to support embedding of fields in an editable rich content area?

**Comment by makenosound** [comment]
@timriley Yep, I reckon we’d need to be able to do that. We’d potentially also want to pass an options hash for each field type alongside an individual content field — so you can control things all the way down.
